recipes-bsp/u-boot/files/imx8m_sign.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
```

Comment:
This script now replaces the script at `dynamic-layers/freescale/recipes-bsp/imx-mkimage/files/mx8m_create_csf.sh`, right?

Should the other script be deleted now or is there a reason to keep it still?

Reply:
Yes, in the end it replaces that script.

Today, as done on a recent PR, we have a deprecation warning saying that the old signing method (based on the output of imx-boot) shouldn't be used and will likely not work. In fact, all the signing is working even with the old method, however the resulting binary does not work (it hangs) simply because the SPL grows to large at least with the default defconfig; though someone eager to make the old method work could try to remove SPL features to downsize it enough to make it boot appropriately. I imagine not many people would be willing to go through that hassle now that we have the new method working.

Considering the above, I think it would be perfectly fine to get rid of the old method even as part of this PR, but I wanted to hear opinions...

Reply:
I think we can drop it as part of this PR. The old implementation will always be there in the kirkstone branch anyways, and we certainly don't want to keep two different implementations here.

Reply:
Thanks guys for the feedback. I'll make this change then.

Reply:
Done on v2.